packages/cli/src/commands/create/CreateCliApp.ts

```diff
@@ -18,16 +18,18 @@ const createAppNft = async (
     connection,
     publisherMintAddress,
     publisher,
+    storageParams,
   }: {
     appDetails: App;
     connection: Connection;
     publisherMintAddress: string;
     publisher: Keypair;
+    storageParams: string;
```

Comment:
(Here and elsewhere) Should this be nullable? What happens if it's not provided as an `option` to the CLI?

Reply:
In `S3StorageManager`, we check to see if the passed in config value is valid. It would seem Javascript is perfectly happy to pass around `undefined` where other types are expected. But it's handled.